### PR TITLE
Simpler checkpoint management

### PR DESCRIPTION
**Pull request by nicolas-chaulet**
This PR changes the behaviour of the checkpoints (hopefully for the better).
Now, models are saved in the hydra directory and models can be loaded from the checkpoint_dir path. Also supports saving and loading the learning rate config.

### CODE REVIEW

File: schedulers/lr_schedulers.py [hunk 1]
```diff
@@ -1,4 +1,9 @@
 from torch.optim.lr_scheduler import LambdaLR
+from omegaconf.dictconfig import DictConfig
+
+
+def build_basic_params():
```
Comment (tchaton) [inline schedulers/lr_schedulers.py]: Maybe, we could use the config.yaml to initialize this one.

Comment (nicolas-chaulet) [reply schedulers/lr_schedulers.py]: Yes, I am not sure how to go about that one...

File: conf/config.yaml [hunk 2]
```diff
@@ -1,16 +1,15 @@
 # Those arguments within experiment defines which model, dataset and task to be created for benchmarking
 experiment:
-    model_name: pointnet2_charlesssg
-    experiment_name: "" # Wether to use hydra naming convention for the experiment
-    log_dir: "" # Wether to use hydra automatic path generation for saving training
+    model_name: pointnet2_charlesmsg
+    checkpoint_dir: "/home/nicolas/deeppointcloud-benchmarks/outputs/2020-01-14/22-56-10"
```
Comment (omry) [inline conf/config.yaml]: you can leave this as null in the config, indicating it's there but not having an actual value.
